hivemq-edge/src/frontend/src/extensions/datahub/components/helpers/PolicyErrorReport.tsx

```diff
@@ -0,0 +1,64 @@
+import { FC } from 'react'
+import {
+  Accordion,
+  AccordionButton,
+  AccordionIcon,
+  AccordionItem,
+  AccordionPanel,
+  Box,
+  Button,
+  ButtonGroup,
+  Text,
+  VStack,
+} from '@chakra-ui/react'
+import { ProblemDetailsExtended } from '@/api/types/http-problem-details.ts'
+import { useTranslation } from 'react-i18next'
+
+interface PolicyErrorReportProps {
+  errors: ProblemDetailsExtended[]
+  onFitView?: (id: string) => void
+  openConfig?: (id: string) => void
+}
+
+const PolicyErrorReport: FC<PolicyErrorReportProps> = ({ errors, onFitView, openConfig }) => {
+  const { t } = useTranslation('datahub')
+  const status = errors.length ? 'warning' : 'success'
+
+  return (
+    <>
+      <Text as="span">{t('workspace.dryRun.report.success.description', { context: status })}</Text>
+      <Accordion allowToggle overflow="auto" maxHeight="180px">
+        {errors.map((problem, i) => {
+          const { id, title, detail } = problem
+          return (
+            <AccordionItem key={`${id as string}-item${i}`} borderColor="blackAlpha.500">
+              <h2>
+                <AccordionButton>
+                  <Box as="span" flex="1" textAlign="left">
+                    {t('workspace.nodes.type', { context: title })}
+                  </Box>
+                  <AccordionIcon />
+                </AccordionButton>
+              </h2>
+              <AccordionPanel pb={4}>
+                <VStack alignItems="flex-start">
+                  <Text>{detail}</Text>
+                  <ButtonGroup alignItems="flex-start" size="sm">
+                    <Button onClick={() => onFitView?.(id as string)}>
+                      {t('workspace.dryRun.report.cta.highlight')}
+                    </Button>
+                    <Button onClick={() => openConfig?.(id as string)}>
```

Comment:
suggestion(non-blocking): we can cast the elements once to prevent doing it multiple times and have one special case within the rendering

```suggestion
          const { id, title, detail } = problem
          
          // TODO: describe why it has to be casted
          const problemId = id as string
          
          return (
            <AccordionItem key={`${problemId}-item${i}`} borderColor="blackAlpha.500">
              <h2>
                <AccordionButton>
                  <Box as="span" flex="1" textAlign="left">
                    {t('workspace.nodes.type', { context: title })}
                  </Box>
                  <AccordionIcon />
                </AccordionButton>
              </h2>
              <AccordionPanel pb={4}>
                <VStack alignItems="flex-start">
                  <Text>{detail}</Text>
                  <ButtonGroup alignItems="flex-start" size="sm">
                    <Button onClick={() => onFitView?.(problemId)}>
                      {t('workspace.dryRun.report.cta.highlight')}
                    </Button>
                    <Button onClick={() => openConfig?.(problemId)}>
```

Reply:
not needed in fact. Not sure where it came from (copy and paste most likely)